### PR TITLE
fix: organization PUT API AttributeError

### LINKED ISSUE #409: PUT `/api/organizations/v0/organizations/{short_name}/` is throwing `AttributeError: This QueryDict instance is immutable`
While calling PUT `/api/organizations/v0/organizations/{short_name}/`, it is throwing `AttributeError: This QueryDict instance is immutable`

Following is API cURL:
```
curl --location --request PUT 'http://local.edly.io:8000/api/organizations/v0/organizations/IITKh/' \
--header 'Authorization: JWT jwt_token' \
--form 'name="IITKh"' \
--form 'short_name="IITKh"'
```

Following is error trace:
```
lms-1  | 2024-09-02 10:35:18,737 ERROR 61 [django.request] [user None] [ip None] log.py:241 - Internal Server Error: /api/organizations/v0/organizations/IITKh/
lms-1  | Traceback (most recent call last):
lms-1  |   File "/openedx/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
lms-1  |     response = get_response(request)
lms-1  |                ^^^^^^^^^^^^^^^^^^^^^
lms-1  |   File "/openedx/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
lms-1  |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
lms-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lms-1  |   File "/opt/pyenv/versions/3.11.8/lib/python3.11/contextlib.py", line 81, in inner
lms-1  |     return func(*args, **kwds)
lms-1  |            ^^^^^^^^^^^^^^^^^^^
lms-1  |   File "/openedx/venv/lib/python3.11/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
lms-1  |     return view_func(*args, **kwargs)
lms-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^
lms-1  |   File "/openedx/venv/lib/python3.11/site-packages/rest_framework/viewsets.py", line 125, in view
lms-1  |     return self.dispatch(request, *args, **kwargs)
lms-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lms-1  |   File "/openedx/venv/lib/python3.11/site-packages/rest_framework/views.py", line 509, in dispatch
lms-1  |     response = self.handle_exception(exc)
lms-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^
lms-1  |   File "/openedx/venv/lib/python3.11/site-packages/rest_framework/views.py", line 469, in handle_exception
lms-1  |     self.raise_uncaught_exception(exc)
lms-1  |   File "/openedx/venv/lib/python3.11/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
lms-1  |     raise exc
lms-1  |   File "/openedx/venv/lib/python3.11/site-packages/rest_framework/views.py", line 506, in dispatch
lms-1  |     response = handler(request, *args, **kwargs)
lms-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lms-1  |   File "/openedx/venv/lib/python3.11/site-packages/organizations/v0/views.py", line 63, in update
lms-1  |     self.request.data['active'] = True
lms-1  |     ~~~~~~~~~~~~~~~~~^^^^^^^^^^
lms-1  |   File "/openedx/venv/lib/python3.11/site-packages/django/http/request.py", line 555, in __setitem__
lms-1  |     self._assert_mutable()
lms-1  |   File "/openedx/venv/lib/python3.11/site-packages/django/http/request.py", line 552, in _assert_mutable
lms-1  |     raise AttributeError("This QueryDict instance is immutable")
lms-1  | AttributeError: This QueryDict instance is immutable
lms-1  | [02/Sep/2024 10:35:18] "PUT /api/organizations/v0/organizations/IITKh/ HTTP/1.1" 500 372660
```

Open edX release: redwood

### PR DESCRIPTION
**Pull request by jramnai**
This fixes #409.
```
self.request.data['active'] = True
```
`self.request.data` is QueryDict hence it is immutable so we can not add `active: True`.

With this PR, `active: True` is added on the serializer side.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @jramnai!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.



#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?


This repository is currently maintained by `@felipemontoya`.  Tag them in a comment and let them know that your changes are ready for review.


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

**Comment by jramnai** [comment]
@felipemontoya, can you please run the workflows? as it requires approval from a maintainer.

**Review by felipemontoya — COMMENTED** [review]
The changes themselves look reasonable to me.

Do you know if this has any implications for other services?

```
Existing organizations updated through the API always end up Active,
regardless of whether or not they were previously active
```

**Comment by jramnai** [comment]
> Do you know if this has any implications for other services?

@felipemontoya, No, I don't know.

**Comment by felipemontoya** [comment]
I did a bit of arqueology on this and the behavior was implemented here:
https://github.com/openedx/edx-organizations/pull/165

As such, this PR does not change the API in any way and downstream services won't be affected.

**Review by felipemontoya — APPROVED** [review]

**Comment by felipemontoya** [comment]
I'll merge the PR now @jramnai. If you need a new tagged version let me know as I was expecting to do some housekeeping on the requirement PRs and add a Changelog before releasing.

**Comment by openedx-webhooks** [comment]
@jramnai 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/edx-organizations&entry.867055334=https://github.com/openedx/edx-organizations/pull/410&entry.1484655318=https://github.com/jramnai&entry.752974735=2024-09-03+10:42&entry.1917517419=2024-09-03+23:44&entry.2133058324=Yes) so we can improve your experience in the future.

**Comment by jramnai** [comment]
Thanks for merging this @felipemontoya. I'm not looking for a new tagged version right now.

**Pull request merged**
